.github/workflows/node.js.yml

```
@@ -5,25 +5,20 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
-    branches: [ main ]
+    branches: [main]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [10.x, 12.x, 14.x]
-
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: yarn install
-    - run: yarn build
-    - run: yarn test
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
```

Since we no longer have a `strategy.matrix` in this action, `${{ matrix.node-version}}` returns nothing, so the default version is used (v16 if I'm not wrong)
```suggestion
      - name: Use Node.js 18
        uses: actions/setup-node@v3
        with:
          node-version: 18
```

---

Committed your suggestion, but this brings OpenSSL issue back, see failed job. It now requires `--openssl-legacy-provider` to be added to `NODE_OPTIONS`.

---

I'm ok with using `openssl-legacy-provider` if needed, I just wanted to know what it is. It seems to be related to [**this issue**](https://github.com/webpack/webpack/issues/14532) with webpack 4 and node 17+

For now, I think the best option is to use that `legacy-provider` arg as you previously did, but in the long term we should upgrade webpack here too.